__tests__/app.test.js

```diff
@@ -41,3 +44,54 @@ describe("GET API", () => {
       });
   });
 });
+
+describe("GET /api/article/:article_id", () => {
+  test("should return a 200 status code", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("should return an object with the correct article with correct keys", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        expect(typeof res.body.article).toBe("object");
+        expect(typeof res.body.article[0].title).toBe("string");
+        expect(typeof res.body.article[0].topic).toBe("string");
+        expect(typeof res.body.article[0].author).toBe("string");
+        expect(typeof res.body.article[0].created_at).toBe("string");
+        expect(typeof res.body.article[0].votes).toBe("number");
+        expect(typeof res.body.article[0].article_img_url).toBe("string");
+        expect(typeof res.body.article[0].article_id).toBe("number");
+      });
+  });
+  test("should return 404 error when passed an article id that doesn't exist", () => {
+    return request(app).get("/api/articles/40").expect(404);
+  });
+});
+
+describe("GET /api/articles", () => {
+  test("should return a 200 status code", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+  test("should return an array of all article objects with the correct keys", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        let count = 0;
+        expect(Array.isArray(res.body.article)).toBe(true);
```

Comment:
the count, this line, and checking the types of every property aren't needed, see my above comments